pkg/reconciler/pipelinerun/pipelinerun_updatestatus_test.go

```diff
@@ -790,7 +790,10 @@ pipelineTaskName: task
 
 			// Sort the ChildReferences to deal with annoying ordering issues.
 			sort.Slice(actualPrStatus.ChildReferences, func(i, j int) bool {
-				return actualPrStatus.ChildReferences[i].PipelineTaskName < actualPrStatus.ChildReferences[j].PipelineTaskName
+				return actualPrStatus.ChildReferences[i].Name < actualPrStatus.ChildReferences[j].Name
```

Comment:
I believe this logic was used because of the complex struct in ChildReferences - SortSlices is useful for simpler types like Strings - what do you think @abayer?

Reply:
`cmpopts.SortSlices` might work, actually, since it takes a function. Worth trying!

Reply:
(I'm gonna do a full followup switching to `cmpopts.SortSlices` everywhere we can in tests)

Reply:
sounds good, thank you! 😁